src/components/new-safe/CreateSafe/useCreateSafe.ts

```diff
@@ -0,0 +1,26 @@
+import type { StepRenderProps } from '@/components/tx/TxStepper/useTxStepper'
+import useIsWrongChain from '@/hooks/useIsWrongChain'
+import useWallet from '@/hooks/wallets/useWallet'
+import { useEffect } from 'react'
+import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import { SAFE_PENDING_CREATION_STORAGE_KEY } from '@/components/create-safe/status/CreationStatus'
+import type { PendingSafeData } from '@/components/create-safe/types.d'
+
+const useCreateSafe = (setStep: StepRenderProps['setStep']) => {
+  const [pendingSafe] = useLocalStorage<PendingSafeData | undefined>(SAFE_PENDING_CREATION_STORAGE_KEY, undefined)
+  const wallet = useWallet()
+  const isWrongChain = useIsWrongChain()
+
+  useEffect(() => {
+    // Jump to the status screen if there is already a tx submitted
+    if (pendingSafe) {
+      setStep(4)
```

Comment:
Currently Step 4 is not included in the stepper which causes an error after I connect a wallet. 
In my local storage there is an empty object `{}` in the localStorage for `pendingSafe` which causes the dialog to always jump to step 4. Probably we should check for `pendingSafe?.txHash` ?

Reply:
Lets remove this part for now until the status step is implemented.